plutus-tx-plugin/src/PlutusTx/Compiler/Expr.hs

```diff
@@ -797,6 +799,18 @@ compileExpr e = traceCompilation 2 ("Compiling expr:" GHC.<+> GHC.ppr e) $ do
     -- Lazy &&
     GHC.App (GHC.App (GHC.Var var) a) b | GHC.getName var == boolOperatorAnd ->
       compileExpr $ GHC.mkIfThenElse a b (GHC.Var GHC.falseDataConId)
+    -- `inline f` or `inline (f args)`
```

Comment:
Nit: I'd perhaps write `f args` as `f x1 ... xn` to make it a bit more clear that it's an iterated application.